popup/app.vue

```diff
@@ -18,25 +18,12 @@
             </ul>
         </div>
         <div id="popup-content" v-else>
-            <div class="row">
-                <div class="col-12">
-                    <h4>Informations générales</h4>
-                </div>
-            </div>
-            <div class="row">
-                <div class="col-12">
-                    <strong>Page :</strong> {{ page.url }}
-                </div>
-            </div>
-            <div class="row">
-                <div class="col-12">
-                    <strong>Auteur :</strong> {{ page.author }}
-                </div>
-            </div>
-            <hr>
             <div class="row">
                 <div class="col-12 text-center">
                     <h4>Niveau de confiance</h4>
+                    <p>
+                        <strong>Page :</strong> {{ page.title }}
```

Comment:
Mettre Titre ou Article à la place de Page me semble plus précis

```suggestion
                        <strong>Titre :</strong> {{ page.title }}
```